src/rollback.cpp

```diff
@@ -483,6 +483,7 @@ const std::list<ActionRow> RollbackManager::actionRowsFromSelect(sqlite3_stmt* s
 		row.actor     = sqlite3_column_int  (stmt, 0);
 		row.timestamp = sqlite3_column_int64(stmt, 1);
 		row.type      = sqlite3_column_int  (stmt, 2);
+		row.nodeMeta  = sqlite3_column_int  (stmt, 8);
```

Comment:
this is probably fine but I think initializing nodeMeta to `0` here and keeping the rest is even better